src/redux/books/booksSlice.js

```diff
@@ -1,36 +1,46 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuidv4 } from 'uuid';
+import { createAsyncThunk } from '@reduxjs/toolkit';
 
-const booksSlice = createSlice({
-  name: 'books',
-  initialState:
-  [
-    {
-      id: 'item1',
-      title: 'The Great Gatsby',
-      author: 'John Smith',
-      category: 'Fiction',
-    },
-    {
-      id: 'item2',
-      title: 'Anna Karenina',
-      author: 'Leo Tolstoy',
-      category: 'Fiction',
-    },
-    {
-      id: 'item3',
-      title: 'The Selfish Gene',
-      author: 'Richard Dawkins',
-      category: 'Nonfiction',
-    },
-  ],
-  reducers: {
-    addBook: (state, action) => {
-      state.push(action.payload);
-    },
-    removeBook: (state, action) => state.filter((book) => book.id !== action.payload),
-  },
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/ymqt9bZZ3hwpTLEEtE1N/books';
+
+// Actions
+export const loadBooks = createAsyncThunk('bookstore/books/LOAD', async () => {
+  const res = await fetch(URL);
+  const json = await res.json();
+  return json;
+});
+
+export const createBook = createAsyncThunk('bookstore/books/CREATE', async (book) => {
+  await fetch(URL, {
+    method: 'POST',
+    body: new URLSearchParams({
+      item_id: uuidv4(),
+      author: book.author,
+      title: book.title,
+      category: book.category,
+    }),
+  });
+  const res = await fetch(URL);
```

Comment:
- [ ] It is required to make all the requests with axios. Kindly consider changing fetch and make use of axios as you have used in the api-content file.